[35mDEBUG   [0m src.steps.common:common.py:27 Running fixture setup: start_2_nodes
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'first_node', '-p', '38141']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2024-12-25T00:51:02.049Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -name first_node
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -port 38141
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2024-12-25T00:51:02.049Z	INFO	first_node	status-cli/util.go:55	starting messenger
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'second_node', '-p', '23553']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - 2024-12-25T00:51:04.060Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -name second_node
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -port 23553
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - 2024-12-25T00:51:04.060Z	INFO	second_node	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [12-25|00:51:04.303] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [12-25|00:51:04.308] New local node record                    seq=1,735,087,864,307 id=029706533d9df0e5 ip=127.0.0.1 udp=0 tcp=43409
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [12-25|00:51:04.308] Started P2P networking                   self="enode://5a8b8dd2f69bc48bb83b899d57c8955156029d0e17f4c3afbce2a46bc1b016c5e0488226749de02e0273a48680d36c8704f82eb2003e4b9934f9e6c08ecb5cfc@127.0.0.1:43409?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - ERROR[12-25|00:51:04.308] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:06.304] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2024/12/25 00:51:06 listen tcp 127.0.0.1:38141: bind: address already in use
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:06.310] New local node record                    seq=1,735,087,866,309 id=03cc425ce02d6f2b ip=127.0.0.1 udp=0 tcp=46217
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:06.310] Started P2P networking                   self="enode://ba09d849a14af81a7fdef222f70ccc30f1cf6c1e08eded3a7347636ec7449dabf07f63fc75481f2b19f76950481f1815fe75b0ce8946db4b0e1462852211d833@127.0.0.1:46217?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - ERROR[12-25|00:51:06.310] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:06.311] HTTP server started                      endpoint=127.0.0.1:23553 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:06.331] New local node record                    seq=1,735,087,866,314 id=f042402fcfe3b26c ip=10.1.0.14 udp=0 tcp=36459
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:06.333] New local node record                    seq=1,735,087,866,315 id=f042402fcfe3b26c ip=10.1.0.14 udp=35608 tcp=36459
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:06.334] New local node record                    seq=1,735,087,866,316 id=f042402fcfe3b26c ip=10.1.0.14 udp=35608 tcp=36459
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - 2024-12-25T00:51:06.767Z	INFO	second_node	status-cli/util.go:68	account created, key UID: 0xf47043e383a0d93b3b073d632fe9f4a57942a0d309e4978852c49b69b9d1fe2f
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:06.767] New local node record                    seq=1,735,087,866,317 id=f042402fcfe3b26c ip=10.1.0.14 udp=35608 tcp=36459
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - 2024-12-25T00:51:06.801Z	INFO	second_node	status-cli/util.go:93	messenger started, public key: 0x04179be3b005bf9d1d1418e064067a81c9c3d96dc5c812af89efd01d99496fedf1c3b6e782ec0e4b06ca3a44293f4b4f2c8e6534daf8fe52fce24e9f10c1e3ff52
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - INFO [12-25|00:51:11.329] New local node record                    seq=1,735,087,866,318 id=f042402fcfe3b26c ip=172.200.180.240 udp=35608 tcp=15296
[35mDEBUG   [0m src.node.status_node:status_node.py:65 SECOND_NODE - 2024-12-25T00:51:11.801Z	INFO	second_node	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: first_node
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: second_node